cmd/kube-dns/app/options/options.go

```diff
@@ -95,9 +97,44 @@ func (m kubeMasterURLVar) Type() string {
 	return "string"
 }
 
+type federationsVar struct {
+	nameDomainMap map[string]string
+}
+
+func (fv federationsVar) Set(v string) error {
```

Comment:
Added a comment/description here which is more relevant to this method. 

Changed the method argument to `keyVal`. It is a Go convention to use short names for method receivers, mostly 1-2 characters long. Also, all other receivers have a single letter names. So leaving this as is.